src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -710,9 +710,24 @@ private String searchRubyHome(Options options) throws IOException {
             return home.getCanonicalPath();
         }
 
-        StringBuilder warning = new StringBuilder("home locations tried:\n");
+        // Use the Truffle reported home
+
+        final String truffleReported = language.getTruffleLanguageHome();
+
+        if (truffleReported != null) {
+            final File home = new File(options.HOME);
```

Comment:
Should this be `new File(truffleReported)`?

Reply:
Yes thanks that was the whole point of this PR!